apps/dashboard/src/app/mint/mint.component.ts

```diff
@@ -1,9 +1,43 @@
 import { Component } from '@angular/core';
+import { FormGroup, FormControl, Validators, FormArray } from '@angular/forms';
+import { COMMA, ENTER } from '@angular/cdk/keycodes';
+import { MatChipInputEvent } from '@angular/material/chips';
 
 @Component({
   selector: 'nft-mint',
   templateUrl: './mint.component.html',
   styleUrls: ['./mint.component.scss'],
 })
 export class MintComponent {
+  readonly separatorKeyCodes = [ENTER, COMMA];
+
+  public mintForm = new FormGroup({
+    name: new FormControl('', Validators.required),
+    image: new FormControl('', Validators.required),
+    jwPlayerId: new FormControl(''),
+    description: new FormControl(''),
+    metadata: new FormGroup({
+      runningTime: new FormControl(null, Validators.required),
+      countries: new FormArray([])
+    })
+  });
+
+  onSubmit() {
+    console.log(this.mintForm);
```

Comment:
work in progress?

Reply:
Yep, I don't do anything after that step